Dockerfile

```diff
@@ -1,5 +1,15 @@
-FROM node:20-alpine
-RUN apk add g++ make python3 dumb-init
+FROM node:20-bullseye
```

Comment:
```suggestion
FROM node:20-bookworm
```
`bookworm` is latest stable version of Debian

Reply:
Might even get away with slim version
```suggestion
FROM node:20-bookworm-slim
```

Reply:
Yep. Slim worked. I also used the RUN command provided by Temporal for slim images.